apps/pie-docs/src/_11ty/shortcodes/mediaElementList.js

```diff
@@ -14,7 +16,7 @@ module.exports = ({ data }) => {
             </picture>
             <div class="c-mediaElement-text">
                 <${hx}>${headingText}</${hx}>
-                <p>${text}</p>
+                <p>${markdownFilter(text)}</p>
```

Comment:
Is this right?
![image](https://user-images.githubusercontent.com/24740015/223771747-fa872a8e-0508-4de6-82ef-bad1fb314320.png)